src/components/EmptyState/EmptyState.tsx

```diff
@@ -15,13 +15,14 @@ import styles from './EmptyState.scss';
 export interface EmptyStateProps {
   /** The empty state heading */
   heading?: string;
-  /** The path to the image to display */
+  /**
+   * The path to the image to display.
+   * The image should have ~40px of white space above when empty state is used within a card, modal, or navigation component
```

Comment:
Is this a recommendation for the formatting of the image used? Or is this noting that ~40px of spacing _will_ be added by the component?

Reply:
We're expecting the image to have white space above. I'll take a look at the best practices now and update 👍